ProjetSupaero2018/catkin_ws/src/controller/scripts/controller_client.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
Do we still need this file ? The car_model is the client now, this file was only for test purpose

Reply:
Yes indeed